websites/C/Crowdin/dist/metadata.json

```diff
@@ -15,24 +15,48 @@
 		"en": "Crowdin is a cloud-based solution that streamlines localization management for your team.",
 		"ro": "Crowdin este o soluție bazată pe cloud care eficientizează gestionarea localizării pentru echipa ta.",
 		"nl": "Crowdin is een cloudgebaseerde oplossing die lokalisatiebeheer voor uw team stroomlijnt.",
-		"vi_VN": "Crowdin là giải pháp trực tuyến cho việc quản lý đội ngũ bản địa hoá của bạn."
+		"vi_VN": "Crowdin là giải pháp trực tuyến cho việc quản lý đội ngũ bản địa hoá của bạn.",
+		"pt": "O Crowdin é uma solução de cloud que facilita a gestão de traduções para a tua equipa.",
+		"ru": "Crowdin — это облачное решение, которое упрощает управление локализацией для вашей команды."
 	},
 	"url": [
-		"translate.premid.app",
 		"crowdin.com",
 		"support.crowdin.com",
 		"store.crowdin.com",
 		"status.crowdin.com",
 		"blog.crowdin.com"
 	],
-	"version": "3.1.2",
+	"regExp": "[a-z]{2}\\.crowdin\\.com",
```

Comment:
`support.crowdin.com` and other urls won't work with this regex, the urls in the array are not used as fallback

Reply:
This is intentional. The regex is not meant to replace the urls array

Reply:
> the urls in the array are not used as fallback

You should see how it works
https://github.com/PreMiD/Extension/blob/release/src/util/tabPriority.ts#L66-L73

Reply:
Now that’s stupid
In that case I won’t support translated Crowdin domains for now

Reply:
removed